smhi/metobs.py

```diff
@@ -102,9 +102,14 @@ def get_data(self, period: str = "corrected-archive"):
         """
         self.data = MetObsDataV1(self.period.period, period)
         self.table_raw = self.data.get()
-        data_starting_point=self.table_raw.find('Datum')
-        self.header=self.table_raw[0:data_starting_point]
-        self.table = pd.read_csv(io.StringIO(self.table_raw[data_starting_point:-1]),sep=';', on_bad_lines="skip",usecols=[0,1,2])
+        data_starting_point = self.table_raw.find("Datum")
+        self.header = self.table_raw[0:data_starting_point]
+        self.table = pd.read_csv(
```

Comment:
Would probably make more sense to return both header and data and not store them inside the object.

Reply:
Fixed in [873649a](https://github.com/Ingenjorsarbete-For-Klimatet/ifk-smhi/pull/4/commits/873649a3a98464167adb88d92667208d67a82926).